Header header
geometry_msgs/Vector3    angular_velocity
geometry_msgs/Vector3    linear_acceleration
geometry_msgs/Vector3    mag
int32                    knob
int32[]                  encoders
int32[]                  forces
uint32[]                 TS
float64		 Trigger_Timestamp

